pow/pow.go

```diff
@@ -17,7 +17,7 @@
 package pow
 
 type PoW interface {
-	Search(block Block, stop <-chan struct{}) (uint64, []byte)
+	Search(block Block, stop <-chan struct{}, index int) (uint64, []byte)
```

Comment:
It's the index of the device used for mining. Couldn't figure out a way to track this without modifying the interface. Open for suggestions of better ways of tracking it - see the ethash PR for how it's used.